cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -343,6 +343,70 @@ def derive_age_columns(df: DataFrame) -> DataFrame:
+            "4-5y and older at school/home-school (including if temporarily absent)": "5y and older in full-time education",  # noqa: E501
+            "5y and older in full-time education": "5y and older in full-time education",  # noqa: E501
+        },
+        "work_status_v2": {
+            "Retired (include doing voluntary work here)": "Retired",  # noqa: E501
+            "Retired": "Retired",  # noqa: E501
+            "Attending college or other further education provider (including apprenticeships) (including if temporarily absent)": "Attending college or FE (including if temporarily absent)",  # noqa: E501
+            "Attending university (including if temporarily absent)": "Attending university (including if temporarily absent)",  # noqa: E501
+            "Child under 5y attending child care": "Child under 4-5y attending child care",  # noqa: E501
+            "Child under 5y attending nursery or pre-school or childminder": "Child under 4-5y attending child care",  # noqa: E501
+            "Child under 4-5y attending nursery or pre-school or childminder": "Child under 4-5y attending child care",  # noqa: E501
+            "Child under 5y not attending nursery or pre-school or childminder": "Child under 4-5y not attending child care",  # noqa: E501
+            "Child under 5y not attending child care": "Child under 4-5y not attending child care",  # noqa: E501
+            "Child under 4-5y not attending nursery or pre-school or childminder": "Child under 4-5y not attending child care",  # noqa: E501
+            "4-5y and older at school/home-school (including if temporarily absent)": "4-5y and older at school/home-school",  # noqa: E501
+            "Employed and currently not working (e.g. on leave due to the COVID-19 pandemic (furloughed) or sick leave for 4 weeks or longer or maternity/paternity leave)": "Employed and currently not working",  # noqa: E501
+            "Employed and currently working (including if on leave or sick leave for less than 4 weeks)": "Employed and currently working",  # noqa: E501
+            "Not in paid work and not looking for paid work (include doing voluntary work here)": "Not working and not looking for work",  # noqa: E501
+            "Self-employed and currently not working (e.g. on leave due to the COVID-19 pandemic or sick leave for 4 weeks or longer or maternity/paternity leave)": "Self-employed and currently not working",  # noqa: E501
+            "Self-employed and currently not working (e.g. on leave due to the COVID-19 pandemic (furloughed) or sick leave for 4 weeks or longer or maternity/paternity leave)": "Self-employed and currently not working",  # noqa: E501
+            "Self-employed and currently working (include if on leave or sick leave for less than 4 weeks)": "Self-employed and currently working",  # noqa: E501
+            "Looking for paid work and able to start": "Looking for paid work and able to start",  # noqa: E501
+        },
+        "work_status": {
```

Comment:
I think this should potentially be work_status_v0 in the pipeline? Worth checking with @howelj